### ISSUE TITLE
Migration from Goodtables to Frictionless Repository

### ISSUE THREAD

**Issue by sapetti9** [augusto-herrmann/transparencia-dados-abertos-brasil#27]
Hey @augusto-herrmann,

Heads up that this repo needs to migrate to Frictionless Repository too before Goodtables is deprecated next year.

https://repository.frictionlessdata.io/

**Comment by augusto-herrmann** [comment]
Yeah, I know. But thanks for the reminder, @sapetti9! :slightly_smiling_face:
